src/offers/client_impls.rs

```diff
@@ -135,6 +137,19 @@ impl InvoicePayer for Client {
         let query_req = tonic_lnd::lnrpc::QueryRoutesRequest {
             amt_msat: msats as i64,
             blinded_payment_paths: vec![blinded_payment_paths],
+            fee_limit: Some(tonic_lnd::lnrpc::FeeLimit {
+                limit: if let Some(fee_limit) = fee_limit {
+                    Some(tonic_lnd::lnrpc::fee_limit::Limit::FixedMsat(
+                        fee_limit as i64,
+                    ))
+                } else if let Some(fee_limit_percent) = fee_limit_percent {
+                    Some(tonic_lnd::lnrpc::fee_limit::Limit::Percent(
+                        (fee_limit_percent * 100.0) as i64,
+                    ))
+                } else {
+                    Some(tonic_lnd::lnrpc::fee_limit::Limit::Percent(100))
```

Comment:
I don't think that we need to force to be this configuration as defaults should be LND's defaults values.
The API should receive `None` in this case